src/manager.cpp

```diff
@@ -30,6 +30,13 @@ sdbusplus::async::task<> Manager::init()
     co_await sdbusplus::async::execution::when_all(
         parseConfiguration(), _extDataIfaces->startExtDataFetches());
 
+    if (_syncBMCDataIface.disable_sync())
```

Comment:
We need to persist this property, but it could be managed as part of the data sync DBus properties persistence work.